.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true,
```

Comment:
@juliusmarminge had an issue where the VSCode TS version was differing from our workspace TS version and showing a type error. This should alert contributors to switch to the workspace version when they first open the project.